apps/wing-console/console/app/.gitattributes

```diff
@@ -0,0 +1 @@
+*.png binary
```

Comment:
Is this really not the default in git?
If not, might as well put this in the root .gitattributes instead.